### ISSUE TITLE
"About Pressbooks" Link in Admin Toolbar has "aria-haspopup"

### ISSUE THREAD

**Issue by avaaec** [pressbooks/pressbooks#1802]
### Steps to Reproduce

1. Go to Admin interface for a book
2. Go to first link in the toolbar labeled "About Pressbooks"
3. Notice that it has "aria-haspopup" attribute
4. Notice that it does not has pop up

**Expected behavior:** 
This link should not has pop up! It does not has pop up.

**Actual behavior:** 
This link has "aria-haspopup"

**Comment by SteelWagstaff** [comment]
@avaaec trying to replicate this, and I see a PB logo in top left of page (admin menu) with html markup as follows:
`<a class="ab-item" aria-haspopup="true" href="https://pressbooks.com/about/"><span class="ab-icon"></span><span class="screen-reader-text">About Pressbooks</span></a>`
When I hover over that link another menu does in fact pop-up (see screenshot below):
![Screenshot from 2019-11-27 19-22-51](https://user-images.githubusercontent.com/13485451/69769654-54bb4580-114b-11ea-92bb-852da5cc5758.png)
Maybe I'm missing something here? Can you clarify?

**Comment by avaaec** [comment]
My apologies @SteelWagstaff when testing this I didn't notice that the button had a drop down menu because the keyboard activation for this button is flawed in the same ways as the other menus in the toolbar (see #1771 ). This button suffers from the same accessibility problems as the other menu buttons in the toolbar. 
We can totally close this issue or fold it into #1771. :)

**Comment by SteelWagstaff** [comment]
Ok -- let's consider this issue closed (as written), but the concern expressed as related to #1771 (which remains open).
